src/filesystem.cpp

```diff
@@ -286,7 +207,7 @@ void FileSystemAccess::escapefsincompatible(string* name, FileSystemType fileSys
             name->replace(i, 1, buf);
             LOG_debug << "Escape incompatible character for filesystem type "
                       << fstypetostring(fileSystemType)
-                      << ", replace '" << std::string(&incompatibleChar, 1) << "' by '" << buf << "'\n";
+                      << ", replace '" << incompatibleChar << "' by '" << buf << "'\n";
```

Comment:
With this change the macro LOG_debug, will print numerical value of incompatibleChar, so you need to preserve the std::string if you want to print the character.